dashboard/app/models/sections/section.rb

```diff
@@ -101,6 +101,10 @@ def workshop_section?
     Pd::Workshop::SECTION_TYPES.include? section_type
   end
 
+  def third_party_login?
+    [LOGIN_TYPE_GOOGLE_CLASSROOM, LOGIN_TYPE_CLEVER].include? login_type
+  end
```

Comment:
Likewise, maybe more future-proof to define this with the inverse set (even though it's currently larger).